Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -159,6 +159,10 @@ void SumSpectra::init() {
                   "If enabled floating point special values such as NaN or Inf"
                   " are removed before the spectra are summed.");
 
+  declareProperty("UseFractionalArea", true,
```

Comment:
We need to be careful when adding new arguments as they can affect the order in the automatically-generated Python function. In this case if a user already has a call to `SumSpectra` with all positional arguments inserting the new argument here will change the meaning of their existing call. Can we add this after `MultiplyBySpectra` ?

Reply:
@Mantid-Matthew Good point I'll push it to the end. Thanks.

Reply:
Moved to the end.